GuillotineMenu/GuillotineMenuViewController.swift

```diff
@@ -89,7 +94,13 @@ class GuillotineMenuViewController: UIViewController {
         }
         
     }
+    
+// MARK: IBAction
 
+    @IBAction func profileButtonTapped(sender: UIButton) {
```

Comment:
I think that we should not handle tapping on buttons in GuillotineMenuViewController. I guess that GuillotineMenuViewController was designed to be very general. We need to redesign the approach and make UI like a separate part of the component and make it as a parameter in the GuillotineMenuViewController. This way those who create UI will responsible for handling events and GuillotineMenuViewController will just add this UI as a child view controller. @maksTheAwesome what do you think ?